src/internal/document_api/mod.rs

```diff
@@ -915,6 +946,100 @@ pub fn decrypt_document<'a, CR: rand::CryptoRng + rand::RngCore>(
         })
 }
 
+/// Decrypt the unmanaged document. The caller must provide both the encrypted data as well as the
+/// encrypted DEKs. Most use cases would want `decrypt_document` instead.
+pub fn decrypt_document_unmanaged<'a, CR: rand::CryptoRng + rand::RngCore>(
```

Comment:
Is there a reason that these methods are named with like `decrypt_document_unmanaged`, but other repos (and the Result classes) are named like `document_decrypt_unmanaged`

Reply:
The main reason is consistency with the already existing `decrypt_document`. The encrypt functions also have this "backwards" naming convention.

I have added this to https://github.com/IronCoreLabs/ironoxide/issues/33